[Bug] generate-effort-report.sh: jq の // 演算子がオブジェクト構築内でシンタックスエラーになる

## 概要

`generate-effort-report.sh` の jq クエリ内で `//`（alternative operator）をオブジェクト構築内で使用した際にシンタックスエラーが発生し、ジョブが失敗する。

## 再現環境

- ワークフロー: `⑤ 統合プロジェクト分析`
- リポジトリ: `lurest-inc/github-projects-starter-kit`
- 実行ログ: https://github.com/lurest-inc/github-projects-starter-kit/actions/runs/23200412054

## エラーメッセージ

```
jq: error: syntax error, unexpected //, expecting '}' (Unix shell quoting issues?) at <top-level>, line 5:
      status: .[0].status // "(未設定)",
jq: 1 compile error
scripts/generate-effort-report.sh: line 213: echo: write error: Broken pipe
##[error]Process completed with exit code 3.
```

## 原因

`scripts/generate-effort-report.sh` 213行目付近のステータス別工数集計処理で、jq のオブジェクト構築 `{}` 内で `//` 演算子を使用している。一部の jq バージョンではオブジェクト構築内の `//` がパーサーに正しく解釈されず、シンタックスエラーとなる。

```jq
# 問題箇所（複数）
status: .[0].status // "(未設定)",
estimated_hours: ([.[].estimated_hours // 0] | add),
actual_hours: ([.[].actual_hours // 0] | add)
```

## 修正方針

`//` 演算子部分を括弧で明示的にグルーピングする。

```jq
# 修正案
status: (.[0].status // "(未設定)"),
estimated_hours: ([.[].estimated_hours // 0] | add),
actual_hours: ([.[].actual_hours // 0] | add)
```

## 関連ファイル

- `scripts/generate-effort-report.sh`（213行目付近）

--- a/scripts/generate-effort-report.sh
+++ b/scripts/generate-effort-report.sh
@@ -197,7 +197,7 @@ STATUS_EFFORT=$(echo "${ITEMS}" | jq --argjson total_estimated "${TOTAL_ESTIMATE
   [.[] | select(.estimated_hours != null or .actual_hours != null)]
   | sort_by(.status // "(未設定)") | group_by(.status // "(未設定)")
   | map({
-      status: .[0].status // "(未設定)",
+      status: (.[0].status // "(未設定)"),
       count: length,
       estimated_hours: ([.[].estimated_hours // 0] | add),
       actual_hours: ([.[].actual_hours // 0] | add)

